docker/slim-dev/Dockerfile

```
@@ -0,0 +1,50 @@
+FROM ubuntu:18.04
+
+LABEL maintainer="Sysdig <support@sysdig.com>"
```

```suggestion
LABEL maintainer="opensource@sysdig.com"
```

The same way we are now doing in the falco builder docker image.